packages/ui-components/src/icon/labicon.tsx

```diff
@@ -829,11 +829,20 @@ namespace Private {
   export function setTitleSvg(svgNode: HTMLElement, title: string): void {
     // add a title node to the top level svg node
     const titleNodes = svgNode.getElementsByTagName('title');
+    // make the title the aria label0
+
+    /// TODO set better unique id for title
     if (titleNodes.length) {
       titleNodes[0].textContent = title;
+      if (!titleNodes[0].id) {
+        titleNodes[0].id = performance.now().toString();
+      }
+      svgNode.setAttribute('aria-labelledby', titleNodes[0].id);
```

Comment:
Looking at https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title, the `title` seems enough to provide an accessible _description_ for a SVG:

![image](https://user-images.githubusercontent.com/8435071/235680018-36743489-3584-4636-bb83-b25aed01c416.png)

Which use case would you like to cover by adding the additional `aria-labelledby`?